db/data_migration/20150514105208_add_eu_logo_url_to_eafrd_editions.rb

```diff
@@ -0,0 +1,24 @@
+slugs = [
+  "countryside-stewardship-facilitation-funding",
+  "countryside-stewardship-water-capital-grants-2015-capital-items",
+  "countryside-stewardship-water-capital-grants-catchment-sensitive-farming",
+  "countryside-stewardship-woodland-capital-grants-2015",
+  "countryside-productivity-scheme",
+  "rural-development-programme-for-england-leader-funding"
+]
+logo_url = "https://assets.digital.cabinet-office.gov.uk/media/55547b94ed915d15d8000057/european-agricultural-fund-for-rural-development.gif"
```

Comment:
Tommy's correct, that's how it was uploaded.

Ideally this would be content managed – something an editor can upload for any document that needs it. Since doing that would involve a bunch more work than we think is worth doing right now, this takes a baby step towards it by making both the image and the relationship to it data, rather than code.